File: 1116.js
let input = '3\n3 -2\n-8 0\n0 8'
let lines = input.split('\n')

let indice = 1

while(indice < lines.length){

    let casos = lines[indice].split(' ')

    let numerador = parseFloat(casos[0])
    let denominador = parseFloat(casos[1]) 

    if(denominador === 0){
        console.log('divisao impossivel')
    }
    else{
        let resultado = numerador/denominador
        if(numerador === 0){
            console.log('0.0')
        }
        else {
            console.log(resultado.toFixed(1))
        }
    }

    indice += 1
}

File: 1116.py
# https://replit.com/languages/python3
quantidade = int(input())

for indice in range(quantidade):
  casos = input().split(' ')
  numerador = float(casos[0])
  denominador = float(casos[1])

  if denominador == 0:
    print('divisao impossivel')
  else:
    resultado = numerador / denominador
    print("{:.1f}".format(resultado))